Brokerages/Exante/ExanteBrokerage.cs

```diff
@@ -0,0 +1,420 @@
+using System.Linq;
+using CryptoExchange.Net.Objects;
+using Exante.Net;
+using Exante.Net.Enums;
+using Exante.Net.Objects;
+using Newtonsoft.Json.Linq;
+using QuantConnect.Data;
+using QuantConnect.Orders.Fees;
+using QuantConnect.Orders.TimeInForces;
+
+namespace QuantConnect.Brokerages.Exante
+{
+    /// <summary>
+    /// The Exante brokerage
+    /// </summary>
+    public partial class ExanteBrokerage : Brokerage
+    {
+        private bool _isConnected;
+        private readonly ExanteClientWrapper _client;
+        private readonly string _accountId;
+        private readonly IDataAggregator Aggregator;
+        private readonly ExanteSymbolMapper _symbolMapper = new ExanteSymbolMapper();
+        private readonly ConcurrentDictionary<Guid, Order> _orderMap = new ConcurrentDictionary<Guid, Order>();
+        private readonly EventBasedDataQueueHandlerSubscriptionManager _subscriptionManager;
+
+        /// <summary>
+        /// Returns the brokerage account's base currency
+        /// </summary>
+        public override string AccountBaseCurrency => Currencies.USD;
+
+        /// <summary>
+        /// Provides the mapping between Lean symbols and Exante symbols.
+        /// </summary>
+        public ExanteSymbolMapper SymbolMapper => _symbolMapper;
+
+        /// <summary>
+        /// Instance of the wrapper class for a Exante REST API client
+        /// </summary>
+        public ExanteClientWrapper Client => _client;
+
+        private static readonly HashSet<string> SupportedCryptoCurrencies = new HashSet<string>()
+        {
+            "ETC", "MKR", "BNB", "NEO", "IOTA", "QTUM", "XMR", "EOS", "ETH", "XRP", "DCR",
+            "XLM", "ZRX", "BTC", "XAI", "ZEC", "BAT", "BCH", "VEO", "DEFIX", "OMG", "LTC", "DASH"
+        };
+
+        /// <summary>
+        /// Creates a new ExanteBrokerage
+        /// </summary>
+        /// <param name="client">Exante client options to create REST API client instance</param>
+        /// <param name="accountId">Exante account id</param>
+        /// <param name="aggregator">consolidate ticks</param>
+        public ExanteBrokerage(
+            ExanteClientOptions client,
+            string accountId,
+            IDataAggregator aggregator
+            )
+            : base("Exante Brokerage")
+        {
+            _client = new ExanteClientWrapper(client);
+            _accountId = accountId;
+            Aggregator = aggregator;
+            _subscriptionManager = new EventBasedDataQueueHandlerSubscriptionManager();
+            _subscriptionManager.SubscribeImpl += Subscribe;
+            _subscriptionManager.UnsubscribeImpl += Unsubscribe;
+
+            _client.StreamClient.GetOrdersStreamAsync(exanteOrder =>
+            {
+                Order order;
+                if (_orderMap.TryGetValue(exanteOrder.OrderId, out order))
+                {
+                    OnOrderEvent(new OrderEvent(order, DateTime.UtcNow, OrderFee.Zero) // TODO: What's the fee?
```

Comment:
> // TODO: What's the fee?

If Exante does not provide it could directly use the FeeModel instance on the associated security, we've done this for a few brokerages too

Reply:
I asked Exante some time ago. They haven't answered. I'll ask again if they provide.